ui/apps/platform/src/Containers/NetworkGraph/utils/networkGraphUtils.ts

```diff
@@ -59,6 +68,17 @@ export function getNumExternalFlows(
     return numExternalFlows;
 }
 
+export function getNumDeploymentFlows(edges: EdgeModel[], deploymentId: string): number {
+    const numFlows =
+        edges?.reduce((acc, edge) => {
+            if (edge.source === deploymentId || edge.target === deploymentId) {
+                return acc + 1;
+            }
+            return acc;
+        }, 0) || 0;
+    return numFlows;
+}
+
```

Comment:
same with this, you can call `getSourceEdges` and `getTargetEdges` on a node to get them natively. i wonder if it'll be worth just calcuating all this at once before the namespace side panel is rendered and just pass through a differently structured `deployments` array that holds the number of edges for each deployment and have the side panel simply render instead of trying to calculate it itself